src/gui/accelerators.c

```diff
@@ -2811,7 +2811,6 @@ gboolean dt_shortcut_dispatcher(GtkWidget *w, GdkEvent *event, gpointer user_dat
     break;
   case GDK_GRAB_BROKEN:
     if(event->grab_broken.implicit) break;
-  case GDK_WINDOW_STATE:
     event->focus_change.in = FALSE; // fall through to GDK_FOCUS_CHANGE
   case GDK_FOCUS_CHANGE: // dialog boxes and switch to other app release grab
```

Comment:
you have `break` on `GDK_GRAB_BROKEN` path, which means that with only `case GDK_WINDOW_STATE:` line removed, the `event->focus_change.in = FALSE;` line is unreachable.

Reply:
That break is conditional though...

If it is _not_ an implicitly broken grab, we will fall through to focus_change.

Reply:
Ah yes...

One of the reasons why I personally dislike actions on same line as `if` :/

Sorry for my lack of reading comprehension!